HW_25.10/HW_25.10/Decerialize.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace HW_25._10
+{
+    public class Decerialize : IDecerialize
+    {
+        private readonly IConfiguration _config;
+        public List<Subject> Decerialization(string path)
```

Comment:
But methods is better to call like verbs =) So it's better to call method Decerialize()